.github/workflows/release.yml

```diff
@@ -0,0 +1,46 @@
+name: Release
```

Comment:
We already have a action named `Release` it could cause issues https://github.com/Shopify/vscode-ruby-lsp/blob/main/.github/workflows/publish.yml#L1